Phoenix/PHSpace.m

```diff
@@ -223,8 +244,23 @@ - (void)removeWindows:(NSArray<PHWindow *> *)windows {
 }
 
 - (void)moveWindows:(NSArray<PHWindow *> *)windows {
-    CGSMoveWindowsToManagedSpace(
-        CGSMainConnectionID(), (__bridge CFArrayRef)[self identifiersForWindows:windows], self.identifier);
+    CGSConnectionID cid = CGSMainConnectionID();
+    if ([NSProcessInfo canMoveWindowsToManagedSpace]) {
```

Comment:
Let’s reverse this conditional after renaming the method.

Reply:
Done!